File: abc/abc185/a.java
// https://atcoder.jp/contests/abc185/submissions/43221954

import java.util.Scanner;

class Main {
  public static void main(String args[]) {
    Scanner stdIn = new Scanner(System.in);

    int[] a = new int[4];
    int ans = 100;

    for(int i = 0; i < 4; i++) {
      a[i] = stdIn.nextInt();
      if(ans > a[i]) {
        ans = a[i];
      }
    }

    System.out.println(ans);
  }
}

File: abc/abc185/a.cs
// https://atcoder.jp/contests/abc185/submissions/46503305

using System;

public class A
{
  static void Main()
  {
    string[] a = Console.ReadLine().Split(" ");

    int ans = 100;
    for(int i = 0; i < a.Length; i++)
    {
      if(int.Parse(a[i]) < ans)
      {
        ans = int.Parse(a[i]);
      }
    }

    Console.Write(ans);
  }
}

